cli/rocks/extra/hardcoded.lua

```diff
@@ -0,0 +1,29 @@
+local function get_cwd()
```

Comment:
Please add some comments about this file (`hardcoded.lua`).

Reply:
Added